### DISCUSSION TITLE
How does the mod determine which blocks are of the correct type when determining building types?

### DISCUSSION THREAD

**Discussion by Gata-chan** [Luke100000/minecraft-comes-alive#943] [Q&A]
Does it use forge tags ro some other way? I've been serching through source but I couldn't find an answer

For context, what happens if I place a modded chest rather than a vanilla chest, would it count? how do I add certain items to the list of things that "count" as certain blocks when determining buildings?

**Comment by Luke100000** [comment]
Resource locations or  tags when prepended with a hashtag. Your chest for example would not count as I specified a resource location. On purpose because the chests are later used and I don't feel like dealing with modded special chests :)

**Reply by Gata-chan** [reply]
Oh, that's a shame but thank you for clarifying
